app/views/brexit_landing_page/_explainer.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="full-page-width-wrapper">
```

Comment:
Made the explainer section its own thing, previously a similar section was embedded in the header